docs/user/04-metrics.md

```diff
@@ -626,13 +648,17 @@ To detect and fix such situations, check the pipeline status and check out [Trou
 2023-08-29T09:53:07.123Z warn internal/transaction.go:111 Failed to scrape Prometheus endpoint {"kind": "receiver", "name": "prometheus/app-pods", "data_type": "metrics", "scrape_timestamp": 1693302787120, "target_labels": "{__name__=\"up\", instance=\"10.42.0.18:8080\", job=\"app-pods\"}"}
 ```
 
-**Cause**: The workload is not configured to use 'STRICT' mTLS mode. For details, see [Activate Prometheus-based metrics](#4-activate-prometheus-based-metrics).
+**Cause 1**: The workload is not configured to use 'STRICT' mTLS mode. For details, see [Activate Prometheus-based metrics](#_4-activate-prometheus-based-metrics).
 
-**Remedy**: You can either set up 'STRICT' mTLS mode or HTTP scraping:
+**Remedy 1**: You can either set up 'STRICT' mTLS mode or HTTP scraping:
 
 - Configure the workload using “STRICT” mTLS mode (for example, by applying a corresponding PeerAuthentication).
 - Set up scraping through HTTP by applying the `prometheus.io/scheme=http` annotation.
 
+**Cause 2**: The Service definition enabling the scrape with Prometheus annotations does not reveal the application protocol to use in the port definition. For details, see [Activate Prometheus-based metrics](#_4-activate-prometheus-based-metrics).
```

Comment:
```suggestion
**Cause 2**: The Service definition enabling the scrape with Prometheus annotations does not reveal the application protocol to use in the port definition. For details, see [Activate Prometheus-based metrics](#4-activate-prometheus-based-metrics).
```
link is broken with underscore

Reply:
```suggestion
**Cause 2**: The Service definition enabling the scrape with Prometheus annotations does not reveal the application protocol to use in the port definition. For details, see [Activate Prometheus-based metrics](#step-4-activate-prometheus-based-metrics).
```

Reply:
Using the heading: 
`### <a name="_4-activate-prometheus-based-metric"></a>4. Activate Prometheus-Based Metric</a>`
and the link: 
`[Activate Prometheus-based metrics](#_4-activate-prometheus-based-metrics)`
works on both GitHub and Doscify. I don't like this workaround but can't find anything better

Reply:
fixed